### ISSUE TITLE
Missing size function

### ISSUE THREAD

**Issue by lorenzhs** [TooBiased/growt#1]
Your table does not implement a `.size()` function. This is a major issue, preventing widespread adoption.

**Comment by TooBiased** [comment]
If you still want/need a size function checkout the branch sizefkt.
There are two options implemented `element_count_approx()` and `element_count_unsafe()`.
The approximate variant is fast and will never be far from the true value. The unsafe variant,
will give the correct value as long as no handle is operating on the table or no handle is being moved.

Before merging it to the master branch I would have to make sure that adding the additional 
functionality did not slow down other operations. I also have to make sure that moving the 
does not lead to any unwanted behavior.

In short checkout the sizefkt branch!

**Comment by lorenzhs** [comment]
Nice, thanks!

**Comment by lorenzhs** [comment]
Reopening because the size function is sometimes off by one or two

**Comment by TooBiased** [comment]
fixed a few days ago!
